users/users-router.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+const Users = require("./users-model");
+
+const imagesRouter = require("../images/images-router");
+const storiesRouter = require("../stories/stories-router");
+
+const restricted = require("../auth/authenticate-middleware");
```

Comment:
Is there any reason for `restricted` to be imported here when it is already acting as middleware when it is protecting the entire `usersRouter` already?

```js
server.use("/users", restricted, usersRouter);
```